Sources/Models/DetectorType.swift

```diff
@@ -31,12 +31,11 @@ public enum DetectorType: Hashable {
     case phoneNumber
     case url
     case transitInformation
-    case custom(pattern: String)
+    case custom(regex: NSRegularExpression)
 
-    // MARK: - Not supported yet
-
-    //case mention
-    //case hashtag
+    // swiftlint:disable force_try
+    public static let hashtag = DetectorType.custom(regex: try! NSRegularExpression(pattern: "#[a-zA-Z]+", options: []))
+    public static let mention = DetectorType.custom(regex: try! NSRegularExpression(pattern: "@[a-zA-Z]+", options: []))
```

Comment:
Numbers should be allowed for mention and hashtag

Reply:
Ok I’ll add it thanks